README.md

```diff
@@ -1,2 +1,78 @@
+# Overview
+Charmed Ceph is a software-defined storage solution that provides object, block and file storage on commodity hardware.
+
+Charmed Ceph makes it easy to use and deploy [Ceph](https://ceph.com/en/). It provides a simpler way to deliver Ceph to users using a set of scripts called charms that are deployed with [Juju](https://juju.is/).
+
+Corporate users can benefit from the robust and manageable full-suite storage solution, and the ease with which their Ceph clusters can be deployed with Charmed Ceph.
+
+# Repository Contents
+
 # ceph-charms
-Monorepo for ceph charms
+
+This is a monolithic repository, or monorepo, that contains the core charms used in Ceph deployment.
+It also contains other associated charm libraries and utilities that make up the Ceph ecosystem that
+can be deployed via Juju.
+
+These charms used to reside in their own repository in the past. The reason they have been consolidated
```

Comment:
[* suggested rewording to clearly state the problem *] "In the past, each charm resided in its own repository, making changes, especially larger changes that involved multiple components, difficult to make. We decided to consolidate all these charms and their associated components into this single repository to make handling changes easier."

I also suggest moving this paragraph to the end of this section.